src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -281,15 +307,67 @@ const ModalCardDetails = ({
     }
   };
 
-  const total = () => {
-    return Number(amount) + feesAmount / 100;
+  const TransactionDetails = () => {
+    const totalPaymentDollars = formatCurrency(100 * amount + feesAmount);
+    const totalContributionDollars = formatCurrency(
+      100 * (amount + matchAmount)
+    );
+
+    if (
+      purchaseType === ModalPaymentTypes.modalPages.mega_gam &&
```

Comment:
Just a suggestion. Since we're using **purchaseType === ModalPaymentTypes.modalPages.mega_gam** a bunch of times throughout this component, what are your thoughts on setting a constant on top which would be either true or false and then using that variable throughout instead? Might make the page a lil cleaner 

const isMegaGam = (purchaseType === ModalPaymentTypes.modalPages.mega_gam)

Reply:
good idea, will do this!